src/components/Details/index.js

```diff
@@ -16,17 +17,25 @@ export default function Details() {
 		<StyledDetails>
 			<h1>{meal.title}</h1>
 			<Button />
-			<section>
+			<section clasName="ingredients">
```

Comment:
You need this classname anymore? Thsi classname has also a typo.

Reply:
I'm really sorry, I thought I deleted the classNames. I deleted them now.